src/main/drivers/vtx_common.c

```diff
@@ -25,18 +25,23 @@
 #include "platform.h"
 #include "build/debug.h"
 
-#if defined(VTX_COMMON)
+#include "common/time.h"
+
+#include "drivers/vtx_common.h"
+
+#include "fc/config.h"
+
+#include "io/vtx.h"
+#include "io/vtx_string.h"
```

Comment:
OK, we're getting there. Includes of `fc/config.h`, `io/vtx.h` and `io/vtx_string.h` should no longer be required, since they are no longer used in this file.